MathLib/InterpolationAlgorithms/PiecewiseLinearInterpolation.h

```diff
@@ -60,7 +60,26 @@ class PiecewiseLinearInterpolation final
      * set to x_{\min} or x_{\max}.
      * @return The interpolated value.
      */
-    double getValue(double pnt_to_interpolate) const;
+    double getValue(double const pnt_to_interpolate) const;
+
+    /**
+     * \brief Calculates the interpolation of the variable by a known
+     *        value of the piece wise linear curve. The curve must be
+     *        monotonically increase or decrease. The monotonicity can be
+     *        checked after create an instance of this class.
+     *
+     * @param value The variable for a point should be located within the range
+     * \f$[f(x_{\min}), f(x_{\max})]\f$, where \f$x_{\min} = \min_{1 \le j \le n}
+     * x_j\f$ and
+     * \f$x_{\max} = \max_{1 \le j \le n} x_j\f$. Points outside of this
+     * interval are
+     * set to f(x_{\min}) or f(x_{\max}).
+     * @return The interpolated value, \f$x\f$.
+     *
+     * \attention Check the monotonicity of the data beforehand if this function
+     *            is used.
+     */
+    double getInverseValue(double const value) const;
```

Comment:
- instead of `variable`, `supporting point` is used in other documentation in this class. please make it consistent. 
- I guess this function returns `variable` (or `supporting point`), not the value. Current function name is confusing and should be renamed, e.g. `getVariable()` or `getPoint()`? 
- can you add a description why the monotonicity is required? and also what will happen if the function is called for non-monotonic case?